.circleci/config.yml

```diff
@@ -66,18 +71,11 @@ jobs:
           name: Lint
           command: yarn lint:check
           working_directory: packages/<<parameters.package_name>>
-      - when:
-          condition:
-            or:
-              - equal: [ "develop", << pipeline.git.branch >> ]
-              - not:
-                  equal: [ "contracts", <<parameters.package_name>> ]
-          steps:
-            - run:
-                name: Test
-                command: yarn test:coverage
-                no_output_timeout: 20m
-                working_directory: packages/<<parameters.package_name>>
```

Comment:
Since this PR moves a lot of logic related to our library into the contracts package, including the tests, I decided to reenable the contracts tests. I've created [a ticket to look into specifically disabling the tests for the core contracts](https://linear.app/chugsplash/issue/CHU-414/investigate-disabling-unnecessary-tests-in-the-contracts-package-when).